lib/promostandards/client.rb

```diff
@@ -14,11 +14,11 @@ class Client
 
     PRIMARY_IMAGE_PRECEDENCE = ['1006', ['1007', '1001', '2001'], ['1007', '1001'], '1007', ['1001', '2001'], '1001', '1003']
 
-    def initialize(access_id:, password:, product_data_service_url:, media_content_service_url:)
-      @access_id = access_id
-      @password = password
-      @product_data_service_url = product_data_service_url
-      @media_content_service_url = media_content_service_url
+    def initialize(config = {})
```

Comment:
We could still use keyword arguments - 
```suggestion
    def initialize(access_id:, password: nil, product_data_service_url:, media_content_service_url: nil)
```

While you are at it, can you also set the default value for `media_content_service_url` as suggested above? I forgot to do this earlier.

Reply:
Didn't get you.

Reply:
Which part? 
1. We can still use keyword arguments (and not a hash argument `config`) by setting default values for `password` and `media_content_service_url`. I've added a code suggestion above.
2. I had missed making `media_content_serive_url` optional. But this is also there in the code suggestion.
These would essentially make `password` and `media_content_service_url` optional and keeping the others mandatory.